src/components/MeetupList.js

```diff
@@ -246,6 +246,8 @@ const Table = styled.div`
 `
 
 const Item = styled(Link)`
+  flex-direction: row;
```

Comment:
FYI, by default any element that is `display: flex` is also `flex-direction: row`, so this is not necessary.